.github/workflows/integration-tests.yml

```diff
@@ -346,4 +346,5 @@ jobs:
       - name: Run python tests on ROCM
         run: |
           cd python
+          pytest --capture=tee-sys -rfs -vvv -n 8  ./test/unit/language/test_subprocess.py -k test_print
```

Comment:
Is there a reason this needs to be a separate invocation of pytest?  (As opposed to just adding test_subprocess to the list of files in the invocation below.)

Reply:
I'm mostly following how other tests are invoked in this file. Also I found the assertion tests are failing right now so this has a filter.

Reply:
You can filter out the assertion tests with -k "not blah".

The advantage of doing it as a single invocation of pytest is that it should be faster.  Probably should change the other invocations too if possible!

Reply:
TIL. Thanks for the suggestions! Makes sense. I've merged them. I can do the other places later.